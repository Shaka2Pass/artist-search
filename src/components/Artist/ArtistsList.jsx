import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

const ArtistList = ({name}) => (

);

ArtistsList.propTypes = {
  name: PropTypes.string.isRequired
};

export default Artists