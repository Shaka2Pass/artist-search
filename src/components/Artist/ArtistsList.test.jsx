import React from 'react';
import { shallow } from 'enzyme';
import ArtistsList from '../Artist/ArtistsList.jsx';

describe('Artist List component', () => {
  it('matches the ArtistLists component', () =>{
    const artistResults = [
      {
        id: 'a3cb23fc-acd3-4ce0-8f36-1e5aa6a18432',
        name: 'U2',
        disambigutation: 'Irish Rock Band'
      },
      {
        id: '704acdbb-1415-4782-b0b6-0596b8c55e46',
        name: 'U2',
        disambigutation: 'Akiyama, Uni'
      }
    ];

    const wrapper = shallow(<ArtistsList artistResults={artistResults} />);
    expect(wrapper).toMatchSnapshot();
  });
});

