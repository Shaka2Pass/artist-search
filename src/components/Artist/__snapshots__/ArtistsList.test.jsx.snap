// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Artist List component matches the ArtistLists component 1`] = `
<ol>
  <li
    key="a3cb23fc-acd3-4ce0-8f36-1e5aa6a18432"
  >
    <Artist
      disambigutation="Irish Rock Band"
      id="a3cb23fc-acd3-4ce0-8f36-1e5aa6a18432"
      name="U2"
    />
  </li>
  <li
    key="704acdbb-1415-4782-b0b6-0596b8c55e46"
  >
    <Artist
      disambigutation="Akiyama, Uni"
      id="704acdbb-1415-4782-b0b6-0596b8c55e46"
      name="U2"
    />
  </li>
</ol>
`;
